### PR TITLE
fix: Gentoo ebuild and FindAubio.cmake

### PR DESCRIPTION
**Pull request by yoda-jm**
Aubio requires CBLAS (for example OpenCBLAS implementation) and not BLAS, this patch should fix the issue.

### CODE REVIEW

**Comment by yoda-jm** [comment]
PTAL, compilation with selft aubio set to NEVER and ALWAYS was tested locally. The issue might be related to the fact that I have both libblas and libopenblas installed and that it mixes things

File: portage-overlay/games-arcade/performous/performous-9999.ebuild [hunk 1]
```diff
@@ -45,7 +46,7 @@ SLOT="0"
 KEYWORDS="~amd64 ~x86"
 IUSE="midi songs tools webcam"
 
-RDEPEND="dev-cpp/libxmlpp:2.6
+RDEPEND="dev-cpp/libxmlpp:3.0
```
[the remark this answers is not in the document]
Comment (yoda-jm) [reply portage-overlay/games-arcade/performous/performous-9999.ebuild:49]: I think this dependency is misleading in the project as a whole, you can find reference to both 2.6 and 3 in various places (docs & pipelines). Gentoo has even switched to 5 (but kept 2.42 for compatibility reasons

**Review by Lord-Kamina — APPROVED** [review]
LGTM

File: portage-overlay/games-arcade/performous/performous-9999.ebuild (hunk 1 above)
[the remark this answers is not in the document]
Comment (Lord-Kamina) [reply portage-overlay/games-arcade/performous/performous-9999.ebuild:49]: @kt-- LibXML++ 2.6, 3, 4 and 5 are made to be able to work side-by-side I think, and performous supports all of them.

**Pull request merged**